graph/GuidelinesGraph.md

```diff
@@ -166,7 +166,8 @@ Microsoft Graph APIs should support basic query options in conformance with ODat
 |Requirements                                                                                        |
 |----------------------------------------------------------------------------------------------------|
 | :heavy_check_mark: **MUST** support `$select on resource` to enable properties projection. |
-| :ballot_box_with_check: **SHOULD** support `\$filter with eq`, `ne` operations on properties of entities for collections. |
+| :ballot_box_with_check: **SHOULD** support `$expand=navProp1` option for navigation properties of entities. |
```

Comment:
Is this saying should have for:

- `GET /entityTypeCollection/{id}?$expand=navProp1` and/or
- `GET /entityTypeCollection?$expand=navProp1`

The latter is more expensive for some resources, so not sure if that's a should or a nice-to-have